MiniApp/Classes/core/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -86,35 +86,52 @@ internal class MiniAppScriptMessageHandler: NSObject, WKScriptMessageHandler {
     }
 
     private func sendMessageToContact(with callBackId: String, parameters: RequestParameters?) {
-        if let message = parameters?.messageToContact {
-            hostAppMessageDelegate?.sendMessageToContact(message, completionHandler: { result in
-                switch result {
-                case .success(let contact):
-                    let notEmptyId: String? = contact?.trimmingCharacters(in: .whitespacesAndNewlines).isEmpty ?? true ? nil : contact
-                    guard let data = try? JSONEncoder().encode(notEmptyId),
-                          let response = String(data: data, encoding: .utf8) else {
-                        return self.executeJavaScriptCallback(
-                                responseStatus: .onError,
-                                messageId: callBackId,
-                                response: getMiniAppErrorMessage(MiniAppJavaScriptError.unexpectedMessageFormat))
+        if isUserAllowedPermission(customPermissionType: .sendMessage, callbackId: callBackId) {
```

Comment:
We don't need to add the permission for `sendMessageToContact` because the Host App displays a contact picker, so basically that acts as the consent for sending a messsage.

We only need the permission for `sendMessageToContactId` because the Host App will send the message in the background without a confirmation from the user.

Reply:
Understood 👍